src/providers/twitch/api/Helix.hpp

```diff
@@ -189,7 +189,7 @@ class Helix final : boost::noncopyable
 private:
     NetworkRequest makeRequest(QString url, QUrlQuery urlQuery);
 
-    QString clientId;
+    QString hclientId;
```

Comment:
random prefix that doesn't carry any meaning and is simply confusing

Reply:
What should it be called?